pages/index.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <div>
+        <UTable :rows="posts" :columns="columnData" :ui="{ td: { base: 'max-w-[0] truncate' } }">
+            <template #id-data="{ row }">
+                <NuxtLink :to="`post/detail/${row.id}`">{{ row.id }}</NuxtLink>
+            </template>
+            <template #body-data="{ row }">
+                <UTooltip :text="row.body">
+                    {{ row.body }}
+                </UTooltip>
+            </template>
+        </UTable>
```

Comment:
This UX could do a lot better with a Card component, laid out responsively. You could use CSS grid / flexbox.